support/managing-cluster-resources.adoc

```diff
@@ -0,0 +1,13 @@
+[id='managing-cluster-resources']
```

Comment:
Not sure if single quotes work or not, but our [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata) illustrate double quotes:
```suggestion
[id="managing-cluster-resources"]
```

Reply:
acked